package.json

```diff
@@ -35,7 +35,8 @@
     "storybook": "start-storybook -p 6006",
     "test": "npx jest",
```

Comment:
I know this is outside of the scope of this PR, but since Max brought up not needing `npx` for the chromatic command, can we remove `npx` here as well?